app/src/main/java/chat/rocket/android/app/DateTimeHelper.kt

```diff
@@ -35,6 +35,17 @@ object DateTimeHelper {
         }
     }
 
+    /**
+     * Returns a time from a LocalDateTime.
+     *
+     * @param localDateTime The LocalDateTime.
+     * @return The time from a LocalDateTime.
+     */
+    fun getTime(localDateTime: LocalDateTime): String {
```

Comment:
I'd vote to use the JSR-310 backport for time (https://github.com/JakeWharton/ThreeTenABP)

Reply:
Some [java.time](https://docs.oracle.com/javase/8/docs/api/java/time/package-summary.html) APIs are available only in the SDK 23+ :confused:

Yeah! We are already [using](https://github.com/filipedelimabrito/Rocket.Chat.Android/blob/f7f0277fd0a9bcdd6eb4c3406f420537c9d4344e/app/src/main/java/chat/rocket/android/app/DateTimeHelper.kt#L4) the ThreeTenABP super cool lib! ❤️